frontend/src/pages/Profile.tsx

```diff
@@ -135,7 +124,7 @@ const XButton = styled.div`
 export function Profile() {
   const [popup, setPopup] = useState<boolean>(false);
   const [allReferringStaff, setAllReferringStaff] = useState<User[]>([]);
-  const [allHousingLocators, setAllHousingLocators] = useState<User[]>([]);
+  const [_allHousingLocators, setAllHousingLocators] = useState<User[]>([]);
```

Comment:
We can remove this line if we don't need this state right?

Reply:
Done!